10379.xml

```diff
@@ -0,0 +1,113 @@
+the metadata provided by the largest instance id takes precedence.</Description>
+    </Description1>
+    <ObjectID>10379</ObjectID>
+    <ObjectURN>urn:oma:lwm2m:x:10379</ObjectURN>
+    <LWM2MVersion>1.1</LWM2MVersion>
+    <ObjectVersion>1.0</ObjectVersion>
+    <MultipleInstances>Multiple</MultipleInstances>
+    <Mandatory>Optional</Mandatory>
+    <Resources>
+        <Item ID="0">
+            <Name>Linked Sensor</Name>
+            <Operations>R</Operations>
+            <MultipleInstances>Single</MultipleInstances>
+            <Mandatory>Mandatory</Mandatory>
+            <Type>Objlnk</Type>
+            <RangeEnumeration></RangeEnumeration>
+            <Units></Units>
+            <Description>Link to the sensor object instance that this metadata relates to.
+            </Item>
+            <Item ID="1">
+                <Name>Measurement Quality Reason Code</Name>
+                <Operations>R</Operations>
+                <MultipleInstances>Multiple</MultipleInstances>
+                <Mandatory>Optional</Mandatory>
+                <Type>Integer</Type>
+                <RangeEnumeration>0..1024</RangeEnumeration>
+                <Units></Units>
+                <Description>Measurement quality degradation reason reported by a smart sensor. 
+A degraded measurement has Measurement Quality Indicator value 1-3 or Measurement Quality Level < 100.
+0: UNKNOWN Reason is unkown, 1: UNDER_RANGE Measured value is under measurement range, 
+2: OVER_RANGE Measured value is over measurement range, 3: NOISY Measured value seems to change too quickly, 
+4: STUCK_VALUE Measured value seems to change too slowly, 5: STEP_TOO_HIGH Measured value has changed too much, 
+6: CALIBRATION_EXPIRED Sensor calibration has expired, 7: WARMING_UP Sensor is warming up, 
+8: SERVICE_MODE Sensor is under service, 9: HUMAN_INTERFERENCE Human presence near the sensor, 
+10: NATURAL_INTERFERENCE Natural phenomenon or animal presence, 11: CONTAMINATION Contamination of the sensor, 
+12: HW_FAILURE Sensor hardware has failed, 13: LACK_OF_DATA_SAMPLES Data sampling is lossy, 
+14: OUT_OF_OPERATING_RANGE Environment conditions exceeded, 15: AUTO_CALIBRATION Auto calibration in progress, 
+16-64: reserved block, 65-128: vendor specific block #1, 129-255: reserved block, 256-1024: vendor specific block #2</Description>
+</Item>
+<Item ID="6042">
+                    <Name>Measurement Quality Indicator</Name>
+                    <Operations>R</Operations>
+                    <MultipleInstances>Single</MultipleInstances>
+                    <Mandatory>Optional</Mandatory>
+                    <Type>Integer</Type>
+                    <RangeEnumeration>0..23</RangeEnumeration>
+                    <Units></Units>
+                    <Description>Measurement quality indicator reported by a smart sensor. 
+0: UNCHECKED No quality checks were done because they do not exist or can not be applied. 
+1: REJECTED WITH CERTAINTY The measured value is invalid. 
```

Comment:
```suggestion
1: REJECTED_WITH_CERTAINTY The measured value is invalid. 
```

For consistency with resource 1, using underscores in the names makes sense (also for other values here)